.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "eslint.validate": [
+    "html",
+    "javascript"
+  ],
+  "editor.codeActionsOnSave": {
+    "source.fixAll": true
+  },
+}
```

Comment:
this seems like something that doesn't belong in an open source repository

Reply:
the workspace settings in general or the code actions specifically?

Reply:
editor-specific settings - editor config is individual choice, and an OSS project shouldn't be biased in favor of one over another (especially in TC39)

Reply:
I see.  I don't interpret it as bias, just workspace config for other people that may use vs code.  In other projects I use it's common to have dotfiles for multiple different editors/IDEs together in the same project.  People can still use whatever they want; but a common config for each editor is helpful and an anti-bikeshedding tool to some extent.  In any case, I'm fine either way.

Reply:
I'm with @ljharb on this, and also, separately, I find the idea of a project telling my editor that it should be running code fixes on save distasteful - that's not the project's decision to make.

Committing these files to a work project where everyone on the team is using the same editors and workflow makes sense, but not so much for OSS projects where there's many contributors with their own preferences.